ddtrace/contrib/redis/patch.py

```diff
@@ -119,6 +121,20 @@ def unpatch():
                 unwrap(redis.asyncio.cluster.ClusterPipeline, "execute")
 
 
+def _run_redis_command(span, func, args, kwargs):
```

Comment:
it doesn't seem there is much difference between this and the async one, should they both live in redis_utils.py?

Reply:
I think actually reading the PR description would help me out....... you are refactoring only the redis utils right now (IIUC?)

Reply:
That's right. The next step on this, probably in a separate pull request, is to remove the span-specific logic from `contrib/redis/patch.py`.